.github/pull_request_template.md

```diff
@@ -0,0 +1,40 @@
+# Proposed changes
+<!-- Describe the changes proposed in this PR.
+
+Provide good PR descriptions as the project maintainers aren't necessarily
+familiar with the packages you are slicing.
+
+We use conventional commits
+(https://www.conventionalcommits.org/en/v1.0.0/#specification), so if not yet
+specified in your commit messages, make sure you describe the type of change
+being proposed in this PR (i.e. feat, test, fix, ci, chore, docs).
+-->
+
+## Related issues/PRs
+<!-- If any -->
+
+### Forward porting
+<!-- This change MUST also be proposed to all newer, and still supported,
+releases. List the corresponding PRs, or ignore if not applicable. -->
+
+## Testing
+<!-- Provide proof of testing and/or testing instructions, when applicable,
+to help speed up the review process. -->
+
+```bash
+# Example:
+#  chisel cut ... --root <path> <pkg>_<proposed-slice>
+#  sudo chroot <path> <app>
+```
```

Comment:
After #120 is merged, will this be needed? In my opinion, this can be part of the "Additional Context" section.

Reply:
I was thinking about functional tests. It's something our CI won't cover for now. I could change the wording, but I think it would be useful if contributors could provide some reproducible way for us (reviewers) to ensure the resulting slices are functional in a real life scenario (e.g. if someone is slicing nodejs, I could use a sample app and execution instructions to test with). Wdyt?

Reply:
Ah that. Yeah, sure, I am okay with that. Let's keep it then.